package.json

```diff
@@ -19,52 +20,48 @@
     "build": "aegir build",
     "release": "aegir release ",
     "release-minor": "aegir release --type minor ",
-    "release-major": "aegir release --type major ",
-    "coverage": "aegir coverage --timeout 100000",
```

Comment:
Can we still have a coverage script?

Reply:
its on my todo to bring back a cov cmd to aegir for now just use `nyc` just like the travis config

Reply:
I meant as a script in this project

Reply:
done